go.mod

```diff
@@ -32,14 +32,24 @@ require (
 	golang.org/x/crypto v0.6.0
 	golang.org/x/net v0.7.0
 	golang.org/x/sync v0.1.0
-	google.golang.org/grpc v1.53.0
+	google.golang.org/grpc v1.52.3
```

Comment:
grpc downgraded because it was pulling a newer `genproto` version. Rest of the changes (other than the "pinned" block) were made by `go mod tidy`.